lit.cfg

```diff
@@ -78,6 +78,7 @@ config.test_exec_root = lit_config.params.get(
 # target and stick it in here.
 if platform.system() == 'Linux':
     config.target_triple = 'linux'
+    linux_run_arch = subprocess.check_output(["uname", "-m"]).strip().decode("utf-8")
```

Comment:
Rather than hard-coding `x86_64` in the linux test, check what arch the tests are run on and use that.